addons/api/addon/models/host-set.js

```diff
@@ -50,6 +50,14 @@ export default class HostSetModel extends GeneratedHostSetModel {
     return this.compositeType === 'azure';
   }
 
+  /**
+   * True if host catalog plugin type is GCP.
+   * @type {boolean}
+   */
+  get isGCP() {
+    return this.compositeType === 'gcp';
```

Comment:
non-blocking: Since we are adding a new type, should we move to the new way of defining types like we see [here](https://github.com/hashicorp/boundary-ui/blob/main/addons/api/addon/models/host-catalog.js)?

Reply:
ah yes, thanks for the catch, updated it